libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/IncidentAlertImpact.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+import androidx.annotation.StringDef
+
+object IncidentAlertImpact {
+
+    /**
+     * Incident Impact unknown.
+     */
+    const val IMPACT_UNKNOWN = "unknown"
+
+    /**
+     * Incident Impact critical.
+     */
+    const val IMPACT_CRITICAL = "critical"
+
+    /**
+     * Incident Impact major.
+     */
+    const val IMPACT_MAJOR = "major"
+
+    /**
+     * Incident Impact minor.
+     */
+    const val IMPACT_MINOR = "minor"
+
+    /**
+     * Incident Impact low.
+     */
+    const val IMPACT_LOW = "low"
+
+    /**
+     * Incident Alert impact.
+     */
+    @Retention(AnnotationRetention.SOURCE)
+    @StringDef(
```

Comment:
As far as I know, stringdefs are ignored by the Kotlin compiler and are basically ignored. If I'm not mistaken, then we should just remove it.

Reply:
yeah, kotlin cannot work with StringDef yet, but it useful for developers to show a range of possible values

btw java compiler just make warnings, that you can ignore. So in java it's a soft check